pandas/core/indexes/timedeltas.py

```diff
@@ -242,6 +239,38 @@ def _format_native_types(self, na_rep=u'NaT', date_format=None, **kwargs):
 
     total_seconds = wrap_array_method(TimedeltaArray.total_seconds, True)
 
+    @classmethod
+    def _override_div_mod_methods(cls):
```

Comment:
why are you defining new class level api here? if you need it here, wouldn't you also need this for base classes? this seems somewhat arbitrary

Reply:
private --> not API
not used by any other classes --> define just for class that uses it

The alternative is to define the methods directly in the class and then back in `Index._add_numeric_methods_binary` special-case to avoid re-defining the methods for `TimedeltaIndex`.  I actually kind of like this, since a couple of PRs from now we'll be able to stop calling `_add_numeric_methods_binary` for `TimedeltaIndex` at all.

Reply:
well, again, rather than create extra helper functions, just put it where everything else is. if you are then going to remove it in the future that is orthognal. the problem with adding things like this is it is just not obvious to anyone else reading this code why this is here and not in the standard location.

Reply:
OK, I'll put it in base.Index I guess